backend/pkg/database/database.go

```diff
@@ -472,14 +518,16 @@ func initDataBase(config *DBConfig) *gorm.DB {
 
 	setupJoinTables(db)
 
+	// drop views before auto migrate
```

Comment:
this is probably conflict with https://github.com/openclarity/kubeclarity/commit/dbec182096b287131bba6a61331bc88d17070e8e, need to rebase